lib/error.js

```diff
@@ -212,4 +212,15 @@ AppError.invalidContentType = function invalidContentType() {
   });
 };
 
+AppError.invalidScopes = function invalidScopes(scopes) {
+  return new AppError({
+    code: 400,
+    error: 'Invalid scopes',
+    errno: 114,
+    message: 'Scopes for untrusted clients are restricted'
```

Comment:
I'd like to use this error as well for refresh tokens, when you originally have a scope of `profile:email` and then try to narrow to scopes you don't have, like `profile:avatar`. Could the message be more general?